### PR TITLE
mail-client/alpine: add new version 2.22

### PR DESCRIPTION
**Pull request by robert7k**
Signed-off-by: Robert Siebeck <gentoo.2019@r123.de>

### CODE REVIEW

File: mail-client/alpine/alpine-2.22.ebuild [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			--with-ssl-include-dir="${EPREFIX}"/usr/include/openssl
+			--with-ssl-lib-dir="${EPREFIX}"/usr/$(get_libdir)
+			--with-ssl-certs-dir="${EPREFIX}"/etc/ssl/certs
+		)
+	fi
+	econf "${myconf[@]}"
+}
+
+src_compile() {
+	emake -j1 AR=$(tc-getAR)
```
Comment (juippis) [inline mail-client/alpine/alpine-2.22.ebuild]: Is this still true? And in `src_install`? We'd like a comment above with a link to upstream issue/bug about it.

File: mail-client/alpine/alpine-2.22.ebuild [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)
+"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+"
+
+HTML_DOCS=("doc/tech-notes/")
```
Comment (juippis) [inline mail-client/alpine/alpine-2.22.ebuild]: This gets defined, but `einstalldocs` isn't called ever in this ebuild. Do these files get installed? Perhaps you want to tie installing them to `doc` USE flag?

Comment (juippis) [reply mail-client/alpine/alpine-2.22.ebuild]: This is still undone. You can just do: 
```
src_install() {
    use doc && local HTML_DOCS=( "doc/tech-notes/" )
    einstalldocs
...
```

and it should install those files and work like it's supposed to. If you do it this way, remember to delete the `HTML_DOCS` here.

File: mail-client/alpine/alpine-2.22.ebuild (hunk 1 above)
Comment (robert7k) [reply mail-client/alpine/alpine-2.22.ebuild]: Yes, this is still required at both locations.

There is no bug tracker, but I'll get in touch with the maintainer to work on this.

File: mail-client/alpine/alpine-2.22.ebuild (hunk 2 above)
Comment (juippis) [reply mail-client/alpine/alpine-2.22.ebuild]: Why did you nuke the extra docs now? What's wrong with the solution given above?

Comment (robert7k) [reply mail-client/alpine/alpine-2.22.ebuild]: They're still installed, see line 79. I thought my solution was shorter than your proposal.

**Pull request closed without merging**